paddle/fluid/operators/math/cross_entropy.h

```diff
@@ -26,9 +27,12 @@ struct TolerableValue {
   HOSTDEVICE T operator()(const T& x) const {
     PADDLE_ASSERT(std::is_floating_point<T>::value);
     const T kApproInf = 1e20;
+    const T kLogMin = log(std::numeric_limits<T>::min());
 
     if (x == INFINITY) return kApproInf;
-    if (x == -INFINITY) return -kApproInf;
+    // TolerableValue is always used after log. So the kLogMin is a more
+    // appropriate value than kApproInf
+    if (x == -INFINITY) return -kLogMin;
```

Comment:
```c++
std::log(std::numeric_limits<float>::min()) = - 87.3365
std::log(std::numeric_limits<double>::min()) = -708.396
```

Here, change from -1e20 to +87.3365 or +708.396

It seems wrong ?

@panyx0718 @kuke How do you think?

Reply:
The final change LGTM

Reply:
forgot to remove the minus...
fixed